### ISSUE TITLE
Question about the result

### ISSUE THREAD

**Issue by hurricane2018** [JiaRenChang/PSMNet#57]
I test the model (pretrained_model_KITTI2015.tar), get the disparity result:
![image](https://user-images.githubusercontent.com/36166137/41265291-75800bec-6dbf-11e8-8080-0a991984c9ba.png)
It looks like the result is right. However, when I assess the accuracy of the disparity. It has a very large error, 46.5% like this.
![image](https://user-images.githubusercontent.com/36166137/41265295-80a86686-6dbf-11e8-8632-ae70b247115f.png)
Could you guys help me ?
Thanks

**Comment by hurricane2018** [comment]
Hi
 The ground truth data is from the training dataset in folder disp_occ_0

![image](https://user-images.githubusercontent.com/36166137/41324088-8eabdca6-6e80-11e8-8391-79622d0fafaa.png)
I evaluate the computed disparity map using official matlab code from KITTI.

**Comment by hurricane2018** [comment]
I don't know what's wrong with this.

**Comment by kalanityL** [comment]
Can you paste your matlab code here?

**Comment by kalanityL** [comment]
```
disp('======= KITTI 2015 Benchmark Demo =======');
%clear all; close all; dbstop error;

% error threshold
tau = [3 0.05];

% stereo demo
disp('Load and show disparity map ... ');
D_est = disp_read('/mnt/1TB/STEREO_DEPTH/PSMNet/output_new/KittiTrain2015byKittiModel/000000_10.png');
D_gt  = disp_read('/mnt/1TB/STEREO_DEPTH/STEREO_DATASETS/KITTI/kitti2015/data_scene_flow/training/disp_occ_0/000000_10.png');
d_err = disp_error(D_gt,D_est,tau);
D_err = disp_error_image(D_gt,D_est,tau);
figure,imshow([disp_to_color([D_est;D_gt]);D_err]);

title(sprintf('Disparity Error: %.2f %%',d_err*100));
```

**Comment by hurricane2018** [comment]
what is 000000_10.png?

Is it the color disparity map? or the black disparity map?

**Comment by kalanityL** [comment]
PathToNetworkOutput/000000_10.png: 
![000000_10](https://user-images.githubusercontent.com/28892491/41666395-1df5a71a-74b3-11e8-8497-250380e632f1.png)

**Comment by passion3394** [comment]
I have rewritten the disp_to_color.m in Python, welcome issues for somebody else, I will try to make the script running fast.

https://github.com/passion3394/PSMNet_CPU/blob/master/disp_to_color.py
